index.js

```diff
@@ -3209,6 +3209,30 @@
   }
   S.find = def('find', {}, [Pred(a), $.Array(a), $Maybe(a)], find);
 
+  //# partition :: (Applicative f, Foldable f, Semigroup (f b), Semigroup (f c)) => (a -> Either b c) -> f a -> Pair (f b) (f c)
```

Comment:
If we say that `f a` and `f b` Semigroups then maybe just say that `f` is `Alternative`